plugin.xml

```diff
@@ -48,7 +48,7 @@
                 <source url="https://github.com/CocoaPods/Specs.git" />
             </config>
             <pods use-frameworks="true">
-                <pod name="ACPGriffon" spec=">= 1.1.0" />
```

Comment:
In this file line 3 u want to change that version as well right

Reply:
Good catch, I didn't see that